#0625

### python init_data.py执行不成功
原因：本地没有安装redis


### 安装redis
参考：www.cnblogs.com/wangyuyu/p/3786236.html

wget http://download.redis.io/releases/redis-4.0.10.tar.gz
tar -zxvf redis-4.0.10.tar.gz
cd redis-4.0.10.tar-gz

make && make install

find ./  | grep redis.conf

mkdir -p /etc/redis/

cp redis.conf > /etc/redis/redis.conf

将conf文件中daemonize no 中no改为yes[yes指后台运行]

redis-server /etc/redis/redis.conf 

ps -ef| grep redis
netstat -tunpl | grep 6379

vi /etc/rc.local #设置随机启动。添加一行
runserver /etc/redis/redis.conf
